script/foreman-debug

```diff
@@ -415,6 +420,8 @@ if [ "$NOTAR" -eq 0 ]; then
   tar -c ../$(basename $DIR) 2>/dev/null | $COMPRESS > "$TARBALL"
   popd >/dev/null
   qprintf "%s: %s\n\n" "A debug file has been created" "$TARBALL ($(stat -c %s "$TARBALL") bytes)"
+  echo "Removing temporary data - \"$DIR\""
+  rm -rf $DIR
```

Comment:
This must be removed, we clean the directory in the trap section, also we make sure it's only deleted when it's `/var/tmp` otherwise we always keep it just in case user would provide some directory with data of a value.